docs/source/api_examples_source/brainglobe_atlasapi.py

```diff
@@ -0,0 +1,198 @@
+"""
+BrainGlobe Atlas API
+====================
+Using the BrainGlobe Atlas API to fetch and inspect an atlas
+"""
+
+
+# %%
+# Import the Atlas API and find an atlas
+# --------------------------------------
+
+# %%
+# Import the API and some other tools
+
+from brainglobe_atlasapi import BrainGlobeAtlas
+from pprint import pprint # to format printed data nicely
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import cm
```

Comment:
This removes a deprecation warning that's generated as part of the output.

```suggestion
from matplotlib import colormaps as cm
```